homeassistant/components/binary_sensor/nissan_leaf.py

```diff
@@ -0,0 +1,49 @@
+"""
+Plugged In Status Support for the Nissan Leaf
+
+Documentation pending.
+Please refer to the main platform component for configuration details
+"""
+
+import logging
+
+from .. import nissan_leaf as LeafCore
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['nissan_leaf']
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
+    devices = []
+
+    _LOGGER.debug("Adding sensors")
+
+    for key, value in hass.data[LeafCore.DATA_LEAF].items():

```

Comment:
You're not using key, so use `values()` instead of `items()`

Reply:
As you can tell, Python is not my first language :)